index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Peer Learning</title>
+        <link rel="stylesheet" href="./style.css">
+    </head>
+
```

Comment:
Please align your code as expected.

Reply:
Boss, I think you don't see well the code is aligned